hack/testdata/redis-slave-replicaset.yaml

```diff
@@ -1,23 +1,19 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1beta2
 kind: ReplicaSet
 metadata:
   name: redis-slave
-  # these labels can be applied automatically
-  # from the labels in the pod template if not set
-  # labels:
-  #   app: redis
-  #   role: slave
-  #   tier: backend
+  labels:
+    app: redis
+    role: slave
+    tier: backend
 spec:
   # this replicas value is default
   # modify it according to your case
   replicas: 2
-  # selector can be applied automatically
-  # from the labels in the pod template if not set
-  # selector:
-  #   app: guestbook
-  #   role: slave
-  #   tier: backend
+  selector:
+    app: redis
```

Comment:
As selector has to match template labels, i changed `app: guestbook` to `app: redis`.